specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/containerregistry_build.json

```diff
@@ -1904,6 +1904,10 @@
         "logLink": {
           "description": "The link to logs for a run on a azure container registry.",
           "type": "string"
+        },
+        "logArtifactLink": {
+          "description": "The link to logs in registry for a run on a azure container registry.",
+          "type": "string"
```

Comment:
Is this a read-only property that is only used in the response?

Reply:
It is only used in the response.

Reply:
@fengzhou-msft Do you have any question? Could you merge?

Reply:
Can you add `"readOnly": true`? It's causing a breaking change without it.